internal/examples/go.mod

```diff
@@ -12,6 +11,7 @@ require (
 	go.uber.org/yarpc v1.42.1
 	go.uber.org/zap v1.13.0
 	google.golang.org/grpc v1.27.1
+	google.golang.org/protobuf v1.25.0
```

Comment:
Why do we have this reference of golang/protov2? the goal was only to move to `github.com/golang/protobuf 1.3.5` no?

Reply:
I don't see any single usage of `google.golang.org/protobuf` in the yarpc code base. Let me check if this is due to indirect dependency.